ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -34,6 +36,91 @@ module Winbind = struct
     debug "init_service To be implemented in CP-36085"
 end
 
+let net_cmd = "/usr/bin/net"
```

Comment:
We usually add commands like these to `xapi_globs.ml`. See here for an example how to add it: https://github.com/xapi-project/xen-api/commit/273339d5f5ff97edf8dc907f291d7d4a6376094a.
Since xapi can work without AD/winbind this should probably go into `nonessential_executables`, so that XAPI doesn't fail to start if it is missing.

Reply:
I do not find other files need to refer this command, so I put in the this file other than the global file, 
also, the `nonessential_executables` table contains command configurable, while this command not.
but I will follow the suggestion if that is what tool stack team does.